anagram.rb

```diff
@@ -0,0 +1,17 @@
+# An anagram is a word or phrase formed by rearranging the letters of a different word or phrase,
```

Comment:
@pmckernin I agree with these edits 👍🏽. One edit is that "program should **print**" since we aren't returning yet and prefer `true` and `false` instead of `True` and `False` since those are different things in Ruby (the capitalized versions don't exist).

Reply:
`true`, `true`.  Yes returning `true` and `false` is better than printing the string of "Ture", this was my typo.

Reply:
Also wow, I really didn't read what I had originally. I must've just copied and pasted it from somewhere without proof reading.